server/@types/index.d.ts

```diff
@@ -60,6 +60,11 @@ export interface Prison {
   active: boolean
 }
 
+export interface PoliceForce {
```

Comment:
Could we use consistent key names for Court, PoliceForce and LocalDeliveryUnit refdata?

Reply:
Also, this will have been automatically generated by swagger-to-ts, no need to manually add here

Reply:
I'm not clear when that happens; I can try this PR without it.

Reply:
About consistency: agreed that is the right goal.  Of course we have to manage the migration.  We've used e.g. `courtid` and `prisonId` but then `policeForceId` gets a bit long hence just `id` and `name`.  Yes, LDU is different again; I've left a TODO in the code for that but we should IMO have tech tickets for such changes 🤔

Reply:
you run `npm run swagger-to-ts`
https://github.com/ministryofjustice/manage-recalls-ui/blob/main/docs/tests.md#typescript-definitions-generated-from-manage-recalls-api-swagger-endpoint

Reply:
I've created this JIRA for the consistency issue: PUD-1118: Make reference data responses consistent

Reply:
I don't understand how you get to see the difference, in a diff, between my adding this and the result of running `npm run swagger-to-ts `?!

Reply:
When I run `npm run swagger-to-ts` it is just making changes unrelated to my change 🤷‍♂️